### PR TITLE
Doc update

### LINKED ISSUE #4: add information about fork to README
The README file should briefly explain what this forked project is all about, how it relates to Locust, and where it originated.

### PR DESCRIPTION
**Pull request by jmelfi**
This should resolve #4 at the top and part of #3.  #3 needs some finishing work on the document structure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,32 @@
-# Locust
+# Cicada 
 
-[![Build Status](https://secure.travis-ci.org/locustio/locust.png)](http://travis-ci.org/locustio/locust) 
-[![Gitter Chat](https://badges.gitter.im/locustio/locust.png)](https://gitter.im/locustio/locust) 
+## Hard Fork
+Hard fork of [Locust](locust.io) due to lack of maintenance by the original development team as seen by the huge PR and Issue backlog.  This is to ensure that new features can be added to the project, as well as needed enhancements to keep it relevant.  You can view the open issue where this originated from on LocustIO's page [here](https://github.com/locustio/cicada/issues/355)
```
Comment (justiniso) [inline README.md]: As frustrated as we all are with the situation, I would adjust the language a bit to soften the message. What do you think of this?

```
## Cicada vs. Locust

As avid users of Locust, many of us have contributed to the main codebase in the past and hope to continue to do so in the future. We immensely appreciate the hard work the authors have put into it. After communicating with them and watching the pull request and issue backlogs grow, we fear that they will not be able to maintain the project to keep up with the surge in popularity. We decided to fork the project staffed by volunteers to keep up with demand and ensure a consistent team of active maintainers. 

For background on the decision to fork, you can read the full conversation on the [originating issue](https://github.com/locustio/locust/issues/355).
```

Comment (jmelfi) [inline README.md]: @justiniso I updated the language to be a bit more reflective of the situation.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,34 @@
-# Locust
+# Cicada 
 
-[![Build Status](https://secure.travis-ci.org/locustio/locust.png)](http://travis-ci.org/locustio/locust) 
-[![Gitter Chat](https://badges.gitter.im/locustio/locust.png)](https://gitter.im/locustio/locust) 
+## Cicada vs Locust
+As avid users of [Locust](locust.io), many have contributed to the main codebase in the past and desire to continue to contribute in the future.  We appreciate the hard work and dedication the [authors](#locust-authors) have put into the original project.  After communication with them and watching the pull request and issue backlogs grow, we fear that they will not be able to maintain the project to keep up with the surge in popularity.  We decided to create a hard fork from Locust into the "Cicada" project staffed by volunteers to keep up with the demand and ensure a team of consistent, active maintainers.
```
Comment (cgoldberg) [inline README.md]: I don't want to suggest the original maintainers weren't capable of the task... so this wording concerns me a bit: 

> "After communication with them and watching the pull request and issue backlogs grow, we fear that they will not be able to maintain the project to keep up with the surge in popularity.  We decided to create a hard fork..."

How about changing it to something like?

> "After communication with them and watching the pull request and issue backlogs grow, a group of community members discussed a way to maintain and modernize the project.  We decided to create a hard fork..."

File: README.md [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
 
-[![Build Status](https://secure.travis-ci.org/locustio/locust.png)](http://travis-ci.org/locustio/locust) 
-[![Gitter Chat](https://badges.gitter.im/locustio/locust.png)](https://gitter.im/locustio/locust) 
+## Cicada vs Locust
+As avid users of [Locust](locust.io), many have contributed to the main codebase in the past and desire to continue to contribute in the future.  We appreciate the hard work and dedication the [authors](#locust-authors) have put into the original project.  After communication with them and watching the pull request and issue backlogs grow, we fear that they will not be able to maintain the project to keep up with the surge in popularity.  We decided to create a hard fork from Locust into the "Cicada" project staffed by volunteers to keep up with the demand and ensure a team of consistent, active maintainers. 
+ 
+You can view the open issue where this originated from on LocustIO's page [here](https://github.com/locustio/cicada/issues/355).
+
+### Fork Goals
+* To maintain the original feature set
```
Comment (cgoldberg) [inline README.md]: > To maintain the original feature set

I don't necessarily agree with this one.  I def don't want to arbitrarily break compatibilty, but keeping strict backward compatibility with locust might not always be practical as we add new features.

File: README.md [hunk 4]
```diff
@@ -1,25 +1,34 @@
...
+* Improve code syntax consistency
+* Add python 3 support
+* Client requests asynchronous (e.g. not using python-requests)
```
Comment (cgoldberg) [inline README.md]: > Client requests asynchronous (e.g. not using python-requests)

Can we remove this item and add an issue to the tracker so we can discuss?
I think the existing synchronous method for modeling client flows is best.  The underlying framework uses gevent, so it's async at that level already (just not within individual greenlets/users)... and I wouldn't want to remove requests.

File: README.md [hunk 5]
```diff
@@ -1,25 +1,34 @@
...
+# Cicada 
 
-[![Build Status](https://secure.travis-ci.org/locustio/locust.png)](http://travis-ci.org/locustio/locust) 
-[![Gitter Chat](https://badges.gitter.im/locustio/locust.png)](https://gitter.im/locustio/locust) 
+## Cicada vs Locust
+As avid users of [Locust](locust.io), many have contributed to the main codebase in the past and desire to continue to contribute in the future.  We appreciate the hard work and dedication the [authors](#locust-authors) have put into the original project.  After communication with them and watching the pull request and issue backlogs grow, we fear that they will not be able to maintain the project to keep up with the surge in popularity.  We decided to create a hard fork from Locust into the "Cicada" project staffed by volunteers to keep up with the demand and ensure a team of consistent, active maintainers. 
+ 
+You can view the open issue where this originated from on LocustIO's page [here](https://github.com/locustio/cicada/issues/355).
+
+### Fork Goals
```
Comment (cgoldberg) [inline README.md]: I think our goals aren't necessarily clear, and it's premature to list them. How about replacing those Fork Goals listed with a simple statement like: 

> We're starting to accumulate new issues and discuss project goals: https://github.com/cicada-load/cicada/issues

File: README.md (hunk 3 above)
Comment (jmelfi) [inline README.md]: I'll edit this to be more clear.  I wouldn't want to tie us to backwards compatibility forever.

File: README.md (hunk 4 above)
Comment (jmelfi) [inline README.md]: I'll remove this one so it can be discussed.  I think I'm remembering problems from quite a long time ago.

**Comment by jmelfi** [comment]
@cgoldberg @justiniso I've updated the README with your suggestions.  Thanks a ton since I absolutely _suck_ at writing these.

Please let me know if there's any other concerns with this work.

**Comment by justiniso** [comment]
:+1: lgtm! We can continue to iterate.

**Pull request closed without merging**